app/models/application_record.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  require 'csv'
```

Comment:
`import.rb` に記載済みですので不要です。無駄にファイルを読み込まないようにすべきです。

```suggestion
  require 'csv'
```